cirq-core/cirq/circuits/circuit_operation.py

```diff
@@ -508,6 +509,16 @@ def _hash(self) -> int:
     def __hash__(self) -> int:
         return self._hash
 
+    def __getstate__(self) -> Dict[str, Any]:
+        # clear cached hash value when pickling, see #6674
+        state = self.__dict__
+        # cached_property stores value in the property name attribute
+        hash_cache = "_hash"
```

Comment:
nit: I'd suggest calling this `hash_attr` since it is the name of the attribute, not itself a cache. Same thing applies in a number of other `__getstate__` methods below.
```suggestion
        hash_attr = "_hash"
```

Reply:
sounds good, done in c43e6ebfa3da9214f76553e945fdd5ff2fb918a6